client/src/components/Transactions/Cell.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect, useState} from "react";
+import styled from 'styled-components';
+
+
+const Cell = (props) =>{
```

Comment:
I have been advised to 'destructure' props with this format:

const Cell = ({ item, user, anotherPropName }) => { your component code... }

and then you can just use the item, or user, or whatever props that you used by their name:
item.category      instead of:      props.item.category

Probably not essential to be done immediately. I started doing this style this morning after Ian and Elfric left me similar comments to this one. Now I kinda like the destructuring props style